trace-utils/src/trace_utils.rs

```diff
@@ -1,40 +1,22 @@
 // Copyright 2023-Present Datadog, Inc. https://www.datadoghq.com/
 // SPDX-License-Identifier: Apache-2.0
 
-use anyhow::{anyhow, Context};
-use futures::stream::FuturesUnordered;
-use futures::StreamExt;
-use hyper::http::HeaderValue;
-use hyper::{body::Buf, Body, Client, HeaderMap, Method, Response, StatusCode};
+use hyper::{body::Buf, Body};
 use log::{error, info};
-use serde::{Deserialize, Serialize};
 use std::collections::HashMap;
 
+pub use crate::send_data::SendData;
```

Comment:
nitpick: Should we move `trace_utils` to use a `mod.rs` file like we do in the sidecar to handle exporting like this? Feels a bit awkward / indirect like this. (If we want to do this, we can do in a separate PR).